src/main/java/uk/gov/justice/hmpps/nomis/datacompliance/events/publishers/DataComplianceEventPusher.java

```diff
@@ -0,0 +1,13 @@
+package uk.gov.justice.hmpps.nomis.datacompliance.events.publishers;
+
+import uk.gov.justice.hmpps.nomis.datacompliance.events.publishers.dto.DataDuplicateResult;
+import uk.gov.justice.hmpps.nomis.datacompliance.events.publishers.dto.OffenderDeletionComplete;
+import uk.gov.justice.hmpps.nomis.datacompliance.events.publishers.dto.OffenderPendingDeletion;
+import uk.gov.justice.hmpps.nomis.datacompliance.events.publishers.dto.OffenderPendingDeletionReferralComplete;
+
+public interface DataComplianceEventPusher {
```

Comment:
Not sure but I'd consider just calling all of these send and let the parameter speak for the send being called..meh.